src/common/components/build-row/index.js

```diff
@@ -66,7 +66,7 @@ BuildRow.propTypes = {
   buildId:  PropTypes.string,
   buildLogUrl: PropTypes.string,
   architecture: PropTypes.string,
-  colour:  PropTypes.oneOf(['green', 'yellow', 'red', 'grey']),
+  colour:  PropTypes.oneOf(['green', 'yellow', 'blue', 'red', 'grey']),
```

Comment:
What about using the exported values where they're defined? https://github.com/canonical-ols/build.snapcraft.io/blob/master/src/common/components/build-status/index.js#L43